releasenotes/notes/migrate-mp2-info-b1a3ee7320c46053.yaml

```diff
@@ -0,0 +1,22 @@
+features:
+  - |
+    Adds a class to compute the Moller-Plesset 2nd Order corrections. The
+    corresponding double-excitation coefficients are intended to be used as
+    an initial point for the VQE parameters (cluster operator coefficients)
+    when using a UCC ansatz. This should introduce an initial point that is
+    closer to the ground state point, leading to fewer overall evaluations for VQE.
+
+    To instantiate `~qiskit_nature.algorithms.MP2PointGenerator`, it requires the
+    number of spin orbitals, the electronic energy, and the excitation lists
+    from the second-quantized properties returned by the driver.
+
+    If using a :class:`~qiskit_nature.algorithms.VQEUCCfactory`, one can set
+    `initial_point="MP2"` and :class:`~qiskit_nature.algorithms.MP2PointGenerator`
```

Comment:
```suggestion
    `initial_point="MP2"` and :class:`~qiskit_nature.algorithms.MP2InitialPoint`
```

Reply:
Actually this needs some more rephrasing because we dropped the string support on `initial_point`.